downstream/modules/automation-hub/proc-enabling-firewall-services.adoc

```diff
@@ -0,0 +1,26 @@
+[id="proc-enabling-firewall-services"]
+
+= Enabling firewall services
+
+Because of the requirement of using a shared filesystem as part of a highly available {HubNameMain} environment, the following firewall services must be enabled to ensure that the filesystem is successfully mounted.
+
+On each {HubNameMain} node, you must:
```

Comment:
This doesn't have the procedure moddocs structure. Could we add the procedure heading?